src/components/VersionSelect/index.spec.tsx

```diff
@@ -129,4 +139,29 @@ describe(__filename, () => {
 
     expect(root.find(`.${className}`)).toHaveLength(1);
   });
+
+  it('marks versions as disabled when they are not selectable', () => {
+    const versions: ExternalVersionsList = [
+      {
+        ...fakeVersionsListItem,
+        id: 123,
+        channel: 'listed',
+        version: 'v1',
+      },
+      {
+        ...fakeVersionsListItem,
+        id: 456,
+        channel: 'unlisted',
+        version: 'v2',
+      },
+    ];
+
+    const root = render({
+      isSelectable: () => false,
```

Comment:
It would be helpful to add another test that makes sure the right `version` object gets passed to `isSelectable()`. It should also test multiple versions where one gets disabled and the other does not.